build.gradle

```diff
@@ -1,33 +1,17 @@
+plugins {
+}
```

Comment:
Looks like this could be removed

Reply:
indeed, how silly of me to overlook that